recipes/soplex/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "6.0.3":
```

Comment:
Matches the naming on the website

![image](https://user-images.githubusercontent.com/16867443/229632867-24148a7f-efcb-497a-99c0-3e24617d03ad.png)